packages/mandelbrot/assets/scss/components/_file-browser.scss

```diff
@@ -31,36 +18,37 @@
     font-weight: bold;
 }
 
-.select2-selection.select2-selection--single {
-    border-color: $color-frame-border;
-
-    .select2-container--focus & {
-        --border-color: rgb(var(--skin-links));
-
-        outline: none;
-        border-color: $color-link;
-        border-color: var(--border-color, $color-link);
-    }
-}
-
-.select2-results__option {
-    @include font(caption);
-
+.FileBrowser-select {
```

Comment:
I’m wondering if we should extract it as a global `.Select` that we could easily reuse later on if necessary. Would be great it shares as many properties as possible with the search input (colors, radius, focus state, …)

Reply:
I wouldn't oppose that, though looking at how the select label is styled currently, it's highly specific to the file browser. I would leave it as it is right now. We can refactor it out when another usecase pops up in the future.

Reply:
👍